### PR TITLE
Include logging with Kafka

### PR DESCRIPTION
**Pull request by guptad2**
This PR sends logging messages to Kafka instance if provided the environment variable process.env.KAFKA_LOGGING.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -28,8 +28,16 @@ function start(clusterProcessId = 0) {
   .then(() => subscriberInit.init(io, processName));
 }
 
+function startWithKafkaLogging() {
+  initKafkaLoggingProducer().then(() => {
```
Comment (iamigo) [inline index.js]: initKafkaLoggingProducer().then(start).catch((err) => logger.error(err));

File: index.js [hunk 2]
```diff
@@ -28,8 +28,16 @@ function start(clusterProcessId = 0) {
   .then(() => subscriberInit.init(io, processName));
 }
 
+function startWithKafkaLogging() {
```
Comment (iamigo) [inline index.js]: this function name doesn't seem right - the decision about whether to use kafka logging or not is embedded in the function implementation itself, right?

Comment (guptad2) [reply index.js]: Makes sense. Fixed.

File: index.js (hunk 1 above)
Comment (guptad2) [reply index.js]: fixed

**Comment by iamigo** [comment]
@guptad2 this is ready for you to update after merging in Jon's logging PR - thx!

File: test/pubSubStats.js [hunk 3]
```diff
@@ -273,9 +273,9 @@ describe('test/pubSubStats.js >', () => {
       const re1 = /info: activity=pubsub key=bye.world process=MyProcessName subCount=3 subTime=\d+ emitCount=3 emitTime=\d+ clientCount=3 clientTime=\d+ \n/; // jscs:ignore maximumLineLength
       const re2 = /info: activity=pubsub key=hello.world process=MyProcessName subCount=1 subTime=\d+ emitCount=1 emitTime=\d+ clientCount=1 clientTime=\d+ \n/; // jscs:ignore maximumLineLength
       expect(inspect.output).to.be.an('Array');
-      expect(inspect.output).to.have.lengthOf(2);
-      expect(inspect.output[0]).to.match(re1);
-      expect(inspect.output[1]).to.match(re2);
```
Comment (jgraff2) [inline test/pubSubStats.js]: Can you explain why the logging is doubled? Is there a way to prevent that so we don't have to change these tests?

Comment (guptad2) [reply test/pubSubStats.js]: Yea sure, there is just a extra log line that is printed when local logging is on. Will change it.

File: src/util/pubSubStats.js [hunk 4]
```diff
@@ -10,11 +10,8 @@
  * /util/pubSubStats.js
  */
 const debug = require('debug')('refocus:pubsub:elapsed');
-const winston = require('winston');
+const logger = require('../logger');
```
[the remark this answers is not in the document]
Comment (guptad2) [reply src/util/pubSubStats.js]: Yes

File: src/logger.js [hunk 5]
```diff
@@ -0,0 +1,85 @@
+/**
```
[the remark this answers is not in the document]
Comment (guptad2) [reply src/logger.js]: It is the same, sounds like a good idea, will look try to get it working that way.

**Pull request closed without merging**